### PR TITLE
Refactor/delegate

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by mikeslgoh** [comment]
do we need to StyleCop for Awake?

**Comment by m4rtint** [comment]
> do we need to StyleCop for Awake?

What do you mean?

File: Assets/Script/CameraFollower.cs [hunk 1]
```diff
@@ -17,11 +17,21 @@ public class CameraFollower : MonoBehaviour
     /// </summary>
     [SerializeField] private Transform m_target;
 
+    private void Awake()
```
Comment (Gursaanj) [inline Assets/Script/CameraFollower.cs]: Yeah, shouldn't Private void Awake() and Private void OnDestroy()  have the summary Descriptions

/// <summary>
///  Whatever
/// </summary>

Comment (m4rtint) [reply Assets/Script/CameraFollower.cs]: It's a private method, it's supposed to be optional if you want to describe it or not  😄

Comment (mikeslgoh) [reply Assets/Script/CameraFollower.cs]: perfect that's good to know

Comment (Gursaanj) [reply Assets/Script/CameraFollower.cs]: Oh did not know that haha

**Pull request merged**